hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoUpdateHandler.java

```diff
@@ -344,10 +345,10 @@ private static long baseSizeOf(final CryptoUpdateTransactionBody op, final long
      * @param account the {@link Account} to be updated
      * @return the calculated bytes
      */
-    private static long cryptoAutoRenewRb(final Account account) {
+    private static long cryptoAutoRenewRb(@Nullable final Account account) {
         return CRYPTO_ENTITY_SIZES.fixedBytesInAccountRepr()
-                + currentNonBaseBytes(account)
-                + account.numberAssociations() * CRYPTO_ENTITY_SIZES.bytesInTokenAssocRepr();
+                + (account == null ? 0 : currentNonBaseBytes(account))
+                + (account == null ? 0 : account.numberAssociations()) * CRYPTO_ENTITY_SIZES.bytesInTokenAssocRepr();
```

Comment:
How about something like
```java
private static long cryptoAutoRenewRb(@Nullable final Account account) {
    var total = CRYPTO_ENTITY_SIZES.fixedBytesInAccountRepr();
    if (account != null) {
        total += currentNonBaseBytes(account) + (account.numberAssociations() * CRYPTO_ENTITY_SIZES.bytesInTokenAssocRepr());
    }
    return total;
}
``` 

instead of checking if the account is null twice?

Reply:
Make sense. Fixed. Thanks @mhess-swl